src/QueryHandler/QueryHandlerInterface.php

```diff
@@ -20,7 +20,7 @@ interface QueryHandlerInterface
      * @param array $params
      * @return array
      */
-    public function query(Schema $schema, $query, array $params = []): array;
+    public function query(Schema $schema, string|DocumentNode $query, array $params = []): array;
```

Comment:
Not supported in PHP 7.4
```suggestion
    public function query(Schema $schema, $query, array $params = []): array;
```